docs/site/index.html

```diff
@@ -1,513 +1,212 @@
+        <link rel="icon" type="image/x-icon" href="https://slint.dev/assets/img/favicon.svg">
 
-                z-index: 10000;
-                white-space: nowrap;
-
-                display: flex;
-                flex-direction: row;
-                align-items: center;
-            }
-
-            nav#Nav > a#main_logo {
-                height: 85%;
-                padding: 6.5px 0px 6.5px 24px;
-            }
-
-            nav#Nav img {
-                height: 100%;
-            }
-
-            nav#Nav ul {
-                flex-grow: 1;
-
-                display: flex;
-                flex-direction: row;
-                align-items: flex-end;
-                justify-content: flex-end;
-                padding: 16px;
-                margin: 0;
-
-                font-weight: bold;
-                color: var(--SLINT_white);
-                font-size: 16px;
-            }
-
-            nav#Nav li {
-                align-self: center;
-                margin: 0px 0px;
-                padding: 16px;
-                list-style-type: none;
-            }
-
-            nav#Nav a {
-                color: inherit;
-            }
-
-            nav#Nav #Hamburger {
-                display: none;
-            }
-
-            #Header {
-                padding: 0px;
-                margin: 0px;
-                width: 100%;
-                background: var(--SLINT_black);
-                background-image: url("https://slint.dev/new_images/Header.svg");
-                background-repeat: no-repeat;
-                background-size: cover;
-                background-position: center;
-                color: var(--SLINT_white);
-                padding: 4rem 5.3rem 3rem 5.3rem;
-            }
-
-            #Header h2,
-            #Header h1 {
-                font-size: 88px;
-                line-height: 110%;
-                max-width: var(--section-width);
-                margin: auto;
-            }
-            #Header small {
-                display: block;
-                max-width: var(--section-width);
-                margin: auto;
-                font-size: 1rem;
-            }
-
-            #Footer {
-                color: var(--SLINT_white);
-                background: radial-gradient(
-                    165.83% 121.13% at 86.81% -12.61%,
-                    #2c2f36 0%,
-                    #040708 53.92%
-                );
-                display: flex;
-                justify-content: space-around;
-                align-items: center;
-                padding: 4vw;
-                clear: both;
-            }
-
-            #Footer > div {
-                font-size: 16px;
-                display: flex;
-                flex-direction: column;
-                align-items: start;
-            }
+        <!-- ===== All CSS files ===== -->
+        <link rel="stylesheet" href="https://slint.dev/assets/css/style.css?v=1.3.0" />
```

Comment:
How does this `v=1.3.0` works.  
I'd rather we had the version in the filename.  Like we have for demos-v1.css
(actually, it would be nice if demos and this page could share the same CSS)

Reply:
CSS versioning is a trick used to make the browser notice the change on your CSS file and request a new CSS file instead of serving one from the cache.